### DISCUSSION TITLE
How to re-enable the preview pane ?

### DISCUSSION THREAD

**Discussion by lenicyl** [keepassxreboot/keepassxc#8106] [Q&A] [answered]
I just realized that my preview panel is no longer present on my keepass
![image](https://user-images.githubusercontent.com/73347143/171896659-65ec92eb-e0d3-4540-8fe9-adae1dea4d8e.png)

I checked the global menu and it seems to be enabled, but the panel still doesnt show.
How do i make it show again ?

**Comment by droidmonkey** [comment, marked as answer]
View menu. Try toggling it on and off..

**Reply by lenicyl** [reply]
I did try that already, but it still doesnt show up

**Reply by droidmonkey** [reply]
Try dragging it back up from the bottom edge (just above the status bar) or selecting another entry/group.

**Reply by lenicyl** [reply]
> Try dragging it back up from the bottom edge (just above the status bar) or selecting another entry/group.

Thanks that worked :) 
I must have accidentally dragged it down at some point without realizing
